trigger/patch.go

```diff
@@ -90,6 +92,13 @@ func (t *patchTriggers) patchOutcome(sub *event.Subscription) (*notification.Not
 	if t.data.Status != evergreen.PatchSucceeded && t.data.Status != evergreen.PatchFailed {
 		return nil, nil
 	}
+	if sub.Subscriber.Type == event.RunChildPatchSubscriberType {
+		err := finalizeChildPatch(t.patch)
```

Comment:
t.patch is the patch that just finished here. So wouldn't this finalize the parent patch?